CHANGELOG.md

```diff
@@ -38,8 +38,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Features
 
-* [\#8559](https://github.com/cosmos/cosmos-sdk/pull/8559) Added Protobuf compatible secp256r1 ECDSA signatures.
-* [\#8786](https://github.com/cosmos/cosmos-sdk/pull/8786) Enabled secp256r1 in x/auth.
```

Comment:
This changelog entry shouldn't be deleted, maybe a merge error?

Reply:
fixed